src/Tizen.NUI.Components/Controls/RecyclerView/Layouter/GridLayouter.cs

```diff
@@ -46,6 +46,35 @@ public class GridLayouter : ItemsLayouter
         private Timer requestLayoutTimer = null;
         private bool isSourceEmpty;
 
+        /// <inheritdoc/>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected new IGroupableItemSource Source
+        {
+            get
+            {
+                if (colView) return colView.InternalSource;
```

Comment:
Unless the intention of `if (colView)` is to check if the BaseHandle's cptr exists, I think it would be better to fix it like this.
```suggestion
                if (colView != null) return colView.InternalSource;
```
Or just replace the whole getter code to
```
                return colView?.InternalSource;
```

FYI, the code `if (colView)` will call `BaseHandle::operator true`. Please refer here:https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/Common/BaseHandle.cs#L141

Reply:
Thank you. will fix this.